### DISCUSSION TITLE
Simple question: how to change field delimiter CSV output?

### DISCUSSION THREAD

**Discussion by aborruso** [saulpw/visidata#1325] [Q&A] [answered]
Hi,
my input file has `;` as delimiter, and I would like to save it using `,`.

If I open it using `vd --csv-delimiter=";" input.csv`, and then I save it, I have again `;` (and it seems right that it should be so). How to change CSV field separator in output?

Thank you

**Comment by saulpw** [comment]
Ah, in that case you would have to change the option on the sheet, before saving.  Are you trying to do this from the CLI?  That would require replaying a .vd to do this.

**Reply by aborruso** [reply]
Thank you @saulpw, in the sheet option what's the option I must change?

![image](https://user-images.githubusercontent.com/30607/158008257-01d76cb4-22f1-42b1-9de3-1e587ecaa59b.png)

Is there a way to set that the CSV field delimeter must be always `,` in output?

Thank you

**Comment by saulpw** [comment, marked as answer]
For saving as `csv`, set `csv_delimiter`.  There isn't a way to set it so that it's always `,` on output.  However, I think that if you set the option to `;` on the cmdline when loading, it won't be global, so that if you use `g"` to duplicate the sheet, and then save the duplicated sheet, the default `,` should be used.

**Reply by aborruso** [reply]
Thank you @saulpw
